dds_web/api/schemas/user_schemas.py

```diff
@@ -132,10 +139,7 @@ def verify_username(self, value):
 
         if utils.username_in_db(username=value):
             raise marshmallow.ValidationError(
-                message=(
-                    f"The username '{value}' is already taken by another user. "
-                    "Try something else."
-                )
+                message=(f"The username '{value}' is already taken by another user. ")
```

Comment:
compiler says parenthesis are not needed, so
`message=f"The username '{value}' is already taken by another user. "`

Reply:
Yep, it's because it's a shorter message.